### PR TITLE
fix: prepare-script

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
fixes error in package.json

See [docs](https://www.npmjs.com/package/husky) for prepare-script

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\"",
     "lint:fix": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "test": "jest",
-    "prepare": "if test \"$HUSKY\" != \"0\" ; then husky install ; fi",
+    "prepare": "husky install",
```
Comment (eddiejaoude) [inline package.json:19]: The extra check is needed for the Dockerfile, we do not want husky to run when building the container - unfortunately husky needs to be installed to do the check itself

Comment (Cahllagerfeld) [reply package.json:19]: Oh I didnt know that. When I ran the `npm install` a few minutes ago, it threw an error in that script 😅

Comment (eddiejaoude) [reply package.json:19]: oh really 😱 let me test it locally, did I fix 1 thing and break another 🤔

Comment (eddiejaoude) [reply package.json:19]: I just tested it, and it worked for me locally. What was your error?

Comment (Cahllagerfeld) [reply package.json:19]:
```bash
> if test "$HUSKY" != "0" ; then husky install ; fi 

""$HUSKY"" kann syntaktisch an dieser Stelle nicht verarbeitet werden. //Translation see below
npm ERR! code ELIFECYCLE
npm ERR! errno 1        
npm ERR! api@0.4.0 prepare: `if test "$HUSKY" != "0" ; then husky install ; fi`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the api@0.4.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Julian\AppData\Roaming\npm-cache\_logs\2021-05-18T18_30_23_252Z-debug.log
```
The marked spot is translated something like this 
`cannot be processed syntactically at this point`

Comment (eddiejaoude) [reply package.json:19]: Oh 🤔 . What OS are you on?

Comment (Cahllagerfeld) [reply package.json:19]: Win10 :D

Comment (eddiejaoude) [reply package.json:19]: oh ok. I am on Mac, and it works on Linux. I think there was an npm library that could do the conditional check for the environment variable

Comment (eddiejaoude) [reply package.json:19]: I could not find what I was looking forward. Another way might be to disable this for the Dockerfile with 

```
npm install --ignore-scripts
```

Comment (eddiejaoude) [reply package.json:19]: I will test this out now and share what happens 🤓

**Comment by eddiejaoude** [comment]
I am going to close this in favour of #115

**Pull request closed without merging**